src/components/movies/Movie.js

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from "react";
+import { getOneMovie, destroyMovie } from "../../api/fetch";
+import { Link, useParams, useNavigate } from "react-router-dom";
+import "../shows/Show.css";
+import ErrorMessage from "../errors/ErrorMessage";
+
+function Movie() {
+  const [loadingError, setLoadingError] = useState(false);
+  const [movie, setMovie] = useState({});
+  const { id } = useParams();
+  const navigate = useNavigate();
+
+  function handleDelete(id) {
+    destroyMovie(id)
+      .then(() => {
+        navigate("/movies");
+        setLoadingError(true);
+      })
+      .catch((error) => {
+        console.log(error);
+        setLoadingError(false);
+      });
+  }
+  useEffect(() => {
+    getOneMovie(id)
+      .then((results) => {
+        setMovie(results);
+        if (results.id) {
+          setLoadingError(false);
+        } else {
+          setLoadingError(true);
+        }
+      })
+      .catch((error) => {
+        console.log(error);
+        setLoadingError(true);
+      });
+  }, [id]);
+
+  return (
+    <section className="shows-show-wrapper">
+      <h2>{Movie.title}</h2>
```

Comment:
this should be `movie.title`

Reply:
sorry my mistake, will fix it. Thank you @Angiepmunoz